jobserver/management/commands/release.py

```diff
@@ -98,7 +98,7 @@ def handle(
             else:
                 raise
 
-        backend = get_or_maybe_create(create, Backend, {"name": backend_name})
+        backend = get_or_maybe_create(create, Backend, {"slug": backend_name})
         user = get_or_maybe_create(create, User, {"username": username})
```

Comment:
Why is slug better than name?

Reply:
In database terms it's the unique field of the two, in UX terms it avoids having to remember the correct capitalisation of a backend's name!

This is all because I changed the names around recently and missed a couple 😬

Reply:
makes sense